pootle/apps/pootle_project/views.py

```diff
@@ -64,21 +64,24 @@ def get_object(self):
         if not (self.kwargs["dir_path"] or self.kwargs["filename"]):
             return self.project
 
-        regex = (
-            r"^/.*/%s/%s"
+        project_path = (
+            "/%s/%s%s"
             % (self.project.code,
-               self.kwargs['dir_path']))
-
+               self.kwargs['dir_path'],
+               self.kwargs['filename']))
```

Comment:
I am bit concerned about what would happen if filename is not provided. Is it `None` or empty string?